pkg/services/provisioning/provisioning.go

```diff
@@ -171,30 +171,36 @@ type ProvisioningServiceImpl struct {
 }
 
 func (ps *ProvisioningServiceImpl) RunInitProvisioners(ctx context.Context) error {
```

Comment:
does this function need to exist if it's empty? i think previously it was being called from `Run()`

Reply:
this is defined by the `ProvisioningService` interface. I checked and saw that the enterprise implementation is not empty, so I thought it would be safer to just keep this method as part of the interface definition.

Reply:
@filewalkwithme is there a chance that enterprise could hit this issue? i'm wondering if moving the `RunInitProvisioners()` back to inside `Run()` would help avoid that

Reply:
This will occur whenever an init provisioner needs to use a Service that goes through k8s apis (currently Folders and Dashboards), regardless of OSS or Enterprise code